### DISCUSSION TITLE
설치하고 작동시킨지 30분만에 예약 성공했습니다.

### DISCUSSION THREAD

**Discussion by YesManL** [changdoc/naver-vaccine-macro#185] [Show and tell]
감사합니다!!

**Comment by cjhaaei** [comment]
혹시 버전몇으로 하셨나요~??

**Reply by YesManL** [reply]
1.1.1 버전으로 했습니다
예약 성공은 카카오 지도를 통해서 성공했습니다

**Reply by HANUL2MERRY** [reply]
카카오 지도에서 자리 나면 바로 네이버 예약 신청으로 연동되서 마무리되는거지요?

**Reply by YesManL** [reply]
> 카카오 지도에서 자리 나면 바로 네이버 예약 신청으로 연동되서 마무리되는거지요?

네 맞아요 저는 그렇게 성공했네요 ㅎㅎ
